exporting/init_connectors.c

```diff
@@ -65,6 +65,9 @@ int init_connectors(struct engine *engine)
 
             // dispatch the instance worker thread
             uv_thread_create(&instance->thread, connector->worker, instance);
+            char threadname[32];
+            snprintfz(threadname, 31, "EXPORTING-%zu", instance->index);
```

Comment:
Why 32 characters? another good reason for the name length `#define`.

Reply:
Done in `threads.h`:
`#define NETDATA_THREAD_NAME_MAX 15`